mu-plugins/central-hub/tests/phpunit/unit/meta-data/remapCustomFieldsConfig.php

```diff
@@ -0,0 +1,110 @@
+<?php
+/**
+ * Tests for the function remap_custom_fields_config().
+ *
+ * @package     spiralWebDb\centralHub\Tests\Unit\Metadata
+ * @since       1.3.0
+ * @author      Robert Gadon <rgadon107>
+ * @link        https://github.com/rgadon107/cornerstone
+ * @license     GNU-2.0+
+ */
+
+namespace spiralWebDb\centralHub\Tests\Unit\Metadata;
+
+use Brain\Monkey;
+use function spiralWebDB\Metadata\remap_custom_fields_config;
+use spiralWebDb\Cornerstone\Tests\Unit\Test_Case;
+
+/**
+ * Class Tests_RemapCustomFieldsConfig
+ *
+ * @package spiralWebDb\centralHub\Tests\Unit\Metadata
+ * @group   meta-data
+ */
+class Tests_RemapCustomFieldsConfig extends Test_Case {
+
+	/**
+	 * Prepare the test environment before each test.
+	 */
+	protected function setUp() {
+		parent::setUp();
+
+		require_once CENTRAL_HUB_ROOT_DIR . '/src/meta-data/meta-box.php';
+	}
+
+	/**
+	 * Clean up the test environment after each test.
+	 */
+	public function tearDown() {
+		parent::tearDown();
+		$config = [];
+	}
+
+	/**
+	 *  Test remap_custom_fields_config() should return the default remapped config array when the given config is an
+	 *  empty array.
+	 */
+	public function test_should_return_default_remapped_config_array_when_given_config_is_empty_array() {
+		$config          = [
+			'meta_box.events' => [
+				'custom_fields' => [],
+			],
+		];
+		$remapped_config = [
+			'is_single'    => [],
+			'default'      => [],
+			'delete_state' => [],
+			'sanitize'     => [],
+		];
+		Monkey\Functions\expect( 'spiralWebDB\Metadata\remap_custom_fields_config' )
```

Comment:
@rgadon107 You don't want to mock out the function being tested. Rather, you want it to run normally.

Tip: Mock out dependencies within the function that is being tested. With this function, there are no dependent functions to mock.

Reply:
Function mock removed.